theano/gof/cmodule.py

```diff
@@ -153,7 +153,7 @@ def __init__(self, name=None):
 
         self.support_code = []
         self.functions = []
-        self.includes = ["<Python.h>", "<iostream>", '"theano_mod_helper.h"']
+        self.includes = ["<Python.h>", "<iostream>", '"theano_mod_helper.h"',"<vector>"]
```

Comment:
Why do you need to do that?  If it is for a specific Op, add it to that op's c_include().